bundler/spec/install/gems/fund_spec.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+RSpec.describe "bundle install" do
+  context "with gem sources" do
+    context "when gems include a fund URI" do
+      it "should display the plural fund message after installing" do
```

Comment:
Same thing about should vs present style for this spec and the others in this file.